### PR TITLE
[Fix bug 1230887] Add max character limit in search form fields

### PR DESCRIPTION
**Pull request by Ladeia**
I add maxlength attribute in all html search forms, and I Django search form.

### CODE REVIEW

**Comment by johngian** [comment]
Hey @Ladeia thanks for your contribution!
Here are some comments regarding your PR:

* Please squash your commits to a single one
    * For more information you can check
       * [git squash tutorial](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
       * [git-rebase docs](https://git-scm.com/docs/git-rebase)
* Use a commit message similar to:
 * ``[Fix bug 1230887] Add max character limit in search fields``

File: mozillians/templates/phonebook/betasearch.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <h1>{{ _('Beta Search') }}</h1>
   <form method="GET" id="search-form" action="{{ url('phonebook:betasearch') }}">
     {{ field_with_attrs(search_form.q,
-                        placeholder=_('Search for people, groups and more')) }}
+                        placeholder=_('Search for people, groups and more'), maxlength="140") }}
```
Comment (johngian) [inline mozillians/templates/phonebook/betasearch.html]: There is no need to add the ``maxlength`` attribute explicitly here. When you are rendering a django form field with specified ``max_length`` it is also render in the html code.

**Comment by johngian** [comment]
Hey @Ladeia ! Is there any update on that PR?
Feel free to ping me if you need any help.

**Comment by Ladeia** [comment]
Sorry @johngian 

I dont use rebese, I have some problems to make it.

I think re-clone the project and remake these changes.

**Pull request closed without merging**